jdat_notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -41,19 +41,53 @@
     "[3] Run ImPlaneIA ([Greenbaum, A. et al. 2015](https://ui.adsabs.harvard.edu/abs/2015ApJ...798...68G/abstract)) to extract observables in oifits format.    "
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #datafiles = sorted(glob.glob(mirage_sim_dir + 'jw*uncal.fits'))
</pre><p>Not sure if these lines are intended for use or should be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>